src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1301,6 +1301,11 @@ public int Compare(Symbol fst, Symbol snd)
                                     Debug.Assert(best.IsFromCompilation);
                                     reportError = false;
                                 }
+                                else if (this.Flags.Includes(BinderFlags.IgnoreCorLibraryDuplicatedTypes) &&
+                                    secondBest.IsFromCorLibrary)
```

Comment:
Naïve question:  Is ```best``` ever "from CorLibrary" when we get to this point?  If not, can we assert that "!best.IsFromCorLibrary"?